### ISSUE TITLE
deprecated-url-parameters rule fires on Livewire ->set() calls in test files

### ISSUE THREAD

**Issue by Yezper** [LaravelDaily/FilaCheck#41]
## Description

The `deprecated-url-parameters` rule triggers false positives when `tableFilters`, `tableSearch` (and related keys) appear as Livewire **wire property names** in `->set()` calls inside Pest/PHPUnit test files. These are not URL query-string parameters — they are the internal Livewire component properties that Filament's table component exposes for testing.

## Steps to reproduce

Write a Filament table filter test using the Livewire testing helpers:

```php
// tests/Feature/SomeFilterTest.php
livewire(ListEmployees::class)
    ->set('tableFilters.my_filter', ['value' => 'foo'])
    ->assertCanSeeTableRecords([$record]);

livewire(ListEmployees::class)
    ->set('tableSearch', 'some query')
    ->assertSeeText('Expected');
```

Run filacheck:

```
vendor/bin/filacheck --fix
```

## Actual behaviour

filacheck flags `tableFilters` and `tableSearch` as deprecated URL parameters and rewrites them to `filters` and `search`:

```
✓ deprecated-url-parameters (Deprecated Code) (fixed)
  tests/Feature/SomeFilterTest.php
    Line 3: The `tableFilters` URL parameter is deprecated. (fixed)
      → Use `filters` instead of `tableFilters`.
    Line 8: The `tableSearch` URL parameter is deprecated. (fixed)
      → Use `search` instead of `tableSearch`.
```

After the rewrite the tests error at runtime:

```
Unable to set component data. Public property [$filters] not found on component: [App\Filament\Resources\Employees\Pages\ListEmployees]
Unable to set component data. Public property [$search] not found on component: [App\Filament\Resources\Employees\Pages\ListEmployees]
```

Because the Livewire component has no `$filters` or `$search` public property — the real wire properties are still `$tableFilters` and `$tableSearch`.

## Root cause

`DeprecatedUrlParametersRule::check()` matches any `String_` AST node whose value contains one of the renamed keys. It does not check whether the string is being used as a URL query-string parameter (inside `$queryString`, `#[Url]`, `->queryStringAlias()`, etc.) versus a Livewire wire property name inside a `->set()` / `->assertSet()` test call.

```php
// src/Rules/DeprecatedUrlParametersRule.php, line 51
if (! str_contains($node->value, $old)) {
    continue;
}
```

## Expected behaviour

The rule should only trigger when the string appears in a URL query-string context, **not** when it is used as a Livewire property name in tests. Possible mitigations:

1. **Scope to known URL contexts** — only flag strings that are keys in `$queryString` arrays, arguments to `#[Url]`, or arguments to `->queryStringAlias()`.
2. **Exclude test files** — skip files under `tests/` entirely, since `->set('tableFilters.…')` is the correct Filament 5 testing API (it's what `filterTable()` uses internally).
3. **Check parent node** — if the string is the first argument to a `->set()` method call, don't flag it.

## Environment

- `laraveldaily/filacheck` — installed via Composer (vendor copy)
- Filament v5
- Livewire v4
- Pest v4

**Comment by PovilasKorop** [comment]
@Yezper thanks for flagging, released new version `v1.2.1`
